scripts/cross.sh

```diff
@@ -68,6 +68,11 @@ case "$TARGETARCH" in
   DPKG_ARCH="s390x"
   PKG_PREFIX="s390x-linux-gnu"
   ;;
+"i386")
```

Comment:
`GOARCH` is `386`. In follow up we should validate binaries architectures. Atm I think no 32bit binaries were built.

Reply:
> Atm I think no 32bit binaries were built.

Yes you're right